packages/components/src/components/PipelineRun/PipelineRun.stories.js

```diff
@@ -0,0 +1,153 @@
+      }
+    ]
+  }
+};
+
+const taskRun = {
+  metadata: { name: 'sampleTaskRunName', namespace: 'default' },
+  spec: {
+    inputs: {},
+    outputs: {},
+    serviceAccount: 'default',
+    taskRef: { kind: 'Task', name: 'task1' },
+    timeout: '24h0m0s'
+  },
+  status: {
+    completionTime: '2019-08-21T17:15:31Z',
+    conditions: [
+      {
+        lastTransitionTime: '2019-08-21T17:15:31Z',
+        message: 'All Steps have completed executing',
+        reason: 'Succeeded',
+        status: 'True',
+        type: 'Succeeded'
+      }
+    ],
+    podName: 'sample-task-run-pod-name',
+    startTime: '2019-08-21T17:12:21Z',
+    steps: [
+      {
+        name: 'build',
+        terminated: {
+          containerID:
+            'docker://88659459cb477936d2ee859822b024bf02768c9ff3dd048f7d8af85843064f95',
+          exitCode: 0,
+          finishedAt: '2019-08-21T17:12:29Z',
+          reason: 'Completed',
+          startedAt: '2019-08-21T17:12:26Z'
+        }
+      }
+    ]
+  }
+};
+
+const pipelineRun = {
+  metadata: {
+    name: 'pipeline-run',
+    namespace: 'cb4552a6-b2d7-45e2-9773-3d4ca33909ff',
+    uid: '7c266264-4d4d-45e3-ace0-041be8f7d06e'
+  },
+  spec: {
+    pipelineRef: {
+      name: 'pipeline'
+    }
+  },
+  status: {
+    conditions: [
+      {
+        lastTransitionTime: '2019-08-16T12:49:28Z',
+        message: 'All Tasks have completed executing',
+        reason: 'Succeeded',
+        status: 'True',
+        type: 'Succeeded'
+      }
+    ],
+    startTime: '2019-08-21T17:12:20Z',
+    taskRuns: {
+      sampleTaskRunName: {
+        pipelineTaskName: 'task1',
+        status: {
+          completionTime: '2019-08-21T17:15:31Z',
+          conditions: [
+            {
+              lastTransitionTime: '2019-08-21T17:15:31Z',
+              message: 'All Steps have completed executing',
+              reason: 'Succeeded',
+              status: 'True',
+              type: 'Succeeded'
+            }
+          ],
+          podName: 'sample-task-run-pod-name',
+          startTime: '2019-08-21T17:12:21Z',
+          steps: [
+            {
+              name: 'build',
+              terminated: {
+                containerID:
+                  'docker://88659459cb477936d2ee859822b024bf02768c9ff3dd048f7d8af85843064f95',
+                exitCode: 0,
+                finishedAt: '2019-08-21T17:12:29Z',
+                reason: 'Completed',
+                startedAt: '2019-08-21T17:12:26Z'
+              }
+            }
+          ]
+        }
+      }
+    }
+  }
+};
+
+storiesOf('PipelineRun', module)
+  .addDecorator(StoryRouter())
+  .add('default', () => (
+    <PipelineRun
+      pipelineRun={pipelineRun}
+      taskRuns={[taskRun]}
+      tasks={[task]}
+      fetchLogs={() => {}}
+      rebuildPipelineRun={() => {}}
+    />
+  ))
+  .add('empty', () => (
+    <PipelineRun
+      pipelineName="Pipeline Name"
+      selectedNamespace="default"
+      cancelPipelineRun={() => {}}
```

Comment:
these props don't look right, I'm not seeing them used in the component
probably ok to remove the 'empty' story since it doesn't really make sense for a PipelineRun
we should probably include at least one of the error cases though